packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/__tests__/LivenessCameraModule.test.tsx

```diff
@@ -69,9 +83,11 @@ describe('LivenessCameraModule', () => {
     isCheckingCamera = false;
     isNotRecording = false;
     isRecording = false;
+    isStart = false;
 
     jest.clearAllMocks();
     jest.clearAllTimers();
+    mockUseLivenessSelector.mockClear();
```

Comment:
Doesn't `jest.clearAllMocks` clear all mocks 😅

Reply:
yep think i added this in frustration while testing lol